### PR TITLE
Convert GetModuleHandle/CreateActCtxW to CsWin33 PInvoke

### PR DESCRIPTION
**Pull request by fernandanavamoya**
Converted these threads:
-GetModuleHandle
-CreateActCtxW

Edited PInvoke.GetModuleHandle.cs file

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/7511)

### CODE REVIEW

**Comment by fernandanavamoya** [comment]
cc: @AArnott

File: src/System.Windows.Forms.Primitives/src/Interop/Kernel32/Interop.LoadLibrary.cs [hunk 1]
```diff
@@ -50,29 +51,29 @@ public static IntPtr LoadComctl32(string startupPath)
         /// </summary>
         /// <param name="libraryName">The assembly name to load.</param>
         /// <returns>A handle to the loaded module, if successful; <see cref="IntPtr.Zero"/> otherwise.</returns>
-        public static IntPtr LoadLibraryFromSystemPathIfAvailable(string libraryName)
+        public static nint LoadLibraryFromSystemPathIfAvailable(string libraryName)
         {
-            IntPtr kernel32 = GetModuleHandleW(Libraries.Kernel32);
-            if (kernel32 == IntPtr.Zero)
+            var kernel32 = PInvoke.GetModuleHandle(Libraries.Kernel32);
+            if (kernel32 == 0)
```
Comment (dreddy-work) [inline src/System.Windows.Forms.Primitives/src/Interop/Kernel32/Interop.LoadLibrary.cs:57]:
```suggestion
            if (PInvoke.GetModuleHandle(Libraries.Kernel32)== 0)
```

File: src/System.Windows.Forms/tests/InteropTests/PropertyGridTests.cs [hunk 2]
```diff
@@ -4,6 +4,9 @@
 
 using System.ComponentModel;
 using System.Runtime.InteropServices;
+using Windows.Win32;
```
Comment (dreddy-work) [inline src/System.Windows.Forms/tests/InteropTests/PropertyGridTests.cs:7]: q: are all these namespaces used?

File: src/System.Windows.Forms.Primitives/src/Interop/Kernel32/Interop.LoadLibrary.cs [hunk 3]
```diff
@@ -50,29 +51,29 @@ public static IntPtr LoadComctl32(string startupPath)
         /// </summary>
         /// <param name="libraryName">The assembly name to load.</param>
         /// <returns>A handle to the loaded module, if successful; <see cref="IntPtr.Zero"/> otherwise.</returns>
-        public static IntPtr LoadLibraryFromSystemPathIfAvailable(string libraryName)
+        public static nint LoadLibraryFromSystemPathIfAvailable(string libraryName)
         {
-            IntPtr kernel32 = GetModuleHandleW(Libraries.Kernel32);
-            if (kernel32 == IntPtr.Zero)
+            var kernel32 = PInvoke.GetModuleHandle(Libraries.Kernel32);
```
Comment (RussKie) [inline src/System.Windows.Forms.Primitives/src/Interop/Kernel32/Interop.LoadLibrary.cs:56]: `var` should only be used when the type is obvious. Please check our [style guide](https://github.com/dotnet/winforms/blob/main/CONTRIBUTING.md#dos-and-donts).

File: src/System.Windows.Forms/tests/InteropTests/PropertyGridTests.cs (hunk 2 above)
Comment (fernandanavamoya) [reply src/System.Windows.Forms/tests/InteropTests/PropertyGridTests.cs:7]: Yes, they are all been used.

File: src/System.Windows.Forms.Primitives/src/Interop/Kernel32/Interop.LoadLibrary.cs (hunk 3 above)
Comment (fernandanavamoya) [reply src/System.Windows.Forms.Primitives/src/Interop/Kernel32/Interop.LoadLibrary.cs:56]: Thank you. Will read the coding guidelines again.

**Review by dreddy-work — APPROVED** [review]

**Pull request merged**